backend/src/chats/chat-user-permissions/chat-user-permission.spec.ts

```diff
@@ -145,60 +134,65 @@ describe('Chat - User - Permission relationship', () => {
 			{
 				chat_id: chat_ids[0],
 				user_id: user_ids[0],
-				permission_id: permission_ids[0],
+				permission: permissionsEnum.READ,
 			},
 			{
 				chat_id: chat_ids[0],
 				user_id: user_ids[1],
-				permission_id: permission_ids[0],
+				permission: permissionsEnum.READ,
 			},
 		]);
 		expect(relations).toHaveLength(2);
 		await service.remove(relations.map((rel) => rel.id));
 	});
 
-	describe('Relations to user/chat entities', () => {
+	describe('Testing "get Users" and "get Chats" methods of user/chat entities', () => {
 		beforeAll(async () => {
 			await service.save([
 				{
 					chat_id: chat_ids[0],
 					user_id: user_ids[0],
-					permission_id: permission_ids[0],
+					permission: permissionsEnum.READ,
 				},
 				{
 					chat_id: chat_ids[0],
 					user_id: user_ids[0],
-					permission_id: permission_ids[1],
+					permission: permissionsEnum.POST,
 				},
 				{
 					chat_id: chat_ids[0],
 					user_id: user_ids[1],
-					permission_id: permission_ids[0],
+					permission: permissionsEnum.READ,
 				},
 				{
 					chat_id: chat_ids[1],
 					user_id: user_ids[0],
-					permission_id: permission_ids[0],
+					permission: permissionsEnum.READ,
 				},
 				{
 					chat_id: chat_ids[2],
 					user_id: user_ids[0],
-					permission_id: permission_ids[2],
+					permission: permissionsEnum.LEFT,
 				},
 				{
 					chat_id: chat_ids[2],
 					user_id: user_ids[1],
-					permission_id: permission_ids[2],
+					permission: permissionsEnum.READ,
 				},
 				{
 					chat_id: chat_ids[2],
 					user_id: user_ids[2],
-					permission_id: permission_ids[1],
+					permission: permissionsEnum.READ,
+				},
+				{
+					chat_id: chat_ids[2],
+					user_id: user_ids[2],
+					permission: permissionsEnum.POST,
 				},
 				{
 					chat_id: chat_ids[1],
 					user_id: user_ids[2],
-					permission_id: permission_ids[0],
+					permission: permissionsEnum.READ,
 				},
```

Comment:
not sure if I understand what happend here? we can also shortly talk about it IRL